keyboards/handwired/wwa/rosetta/info.json

```
@@ -0,0 +1,47 @@
+{
+    "manufacturer": "spacehangover",
+    "keyboard_name": "Rosetta",
+    "maintainer": "spacehangover",
+    "development_board": "promicro",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["C6", "D4", "D0", "D1"],
+        "rows": ["B4", "E6", "D7"]
+    },
+    "processor": "atmega32u4",
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0006",
+        "vid": "0x5757"
+    },
```

migrated from `config.h`
```suggestion
    },
    "qmk": {
        "locking": {
            "enabled": true,
            "resync": true
        }
    },
```